mls-rs/src/group/message_processor.rs

```diff
@@ -284,19 +284,19 @@ pub struct ProposalMessageDescription {
     pub proposal: Proposal,
     /// Plaintext authenticated data in the received MLS packet.
     pub authenticated_data: Vec<u8>,
-    #[cfg(all(feature = "by_ref_proposal", feature = "external_client"))]
+    #[cfg(feature = "by_ref_proposal")]
     pub(crate) proposal_ref: ProposalRef,
 }
```

Comment:
Not related to this PR, but should this struct be non-exhaustive? Should this field be public like the other fields?

Reply:
Good point. The cfg guard is also redundant. I pushed a commit to address all that. Thank you!